style.css

```diff
@@ -0,0 +1,63 @@
+body {
+    padding: 10px 15px;
+    display: flex;
+}
+
+div.input {
+    width: calc(100% - 17em);
+}
+
+div.damageTable {
+    position: fixed;
```

Comment:
Eventually we should probably structure it so that the scrollbar is only on the output tables or whatever, rather than scrolling the whole page and fixing some elements.